components/suggestion/search_date_suggestion/search_date_suggestion.jsx

```diff
@@ -5,10 +5,31 @@ import React from 'react';
 
 import DayPicker from 'react-day-picker';
 
+import MomentLocaleUtils from 'react-day-picker/moment';
+
 import Suggestion from '../suggestion.jsx';
 
 import 'react-day-picker/lib/style.css';
 
+import 'moment';
+
+import 'moment/locale/de';
```

Comment:
Would it be possible to import only the packages here that are actually needed for the user's locale to save a bit of bandwidth? We might be able to do something like
```javascript
const loadedLocales = {};

export default class SearchDateSuggestion extends Suggestion {
    componentDidMount(props) {
        if (props.locale && !loadedLocales[props.locale]) {
            // There might be something special needed here as well since we use `en` instead of
            // `en-ca` for English.
            loadedLocales[props.locale] = require(`moment/locale/${props.locale}`);
        }
    }
    
    componentDidUpdate(prevProps) {
        // do the same thing here if locale changes
    }
}
```

Reply:
Very clever! Will do!

Reply:
I have implemented this change, but it seems lint does not like it. Currently I asked the lint to ignore it, please let me know what to do.

Reply:
That's totally fine. We've done similar things for other libraries that we only want loaded sometimes.